src/rebloom.c

```diff
@@ -83,8 +84,9 @@ static const char *statusStrerror(int status) {
  * Common function for adding one or more items to a bloom filter.
  * capacity and error rate must not be 0.
  */
-static SBChain *bfCreateChain(RedisModuleKey *key, double error_rate, size_t capacity, unsigned expansion) {
-    SBChain *sb = SB_NewChain(capacity, error_rate, BLOOM_OPT_FORCE64, expansion);
+static SBChain *bfCreateChain(RedisModuleKey *key, double error_rate,
+                              size_t capacity, unsigned expansion, unsigned scaling) {
```

Comment:
here scaling is unsigned

Reply:
@gkorland 
Redis does only RedisModule_StringToLongLong but _options_ in _SBChain_ is unsigned.

Reply:
@gkorland 
OK. That was silly.
Fixed